### ISSUE TITLE
Increase Or make Max-Width/Max-Height Configureable?

### ISSUE THREAD

**Issue by tobiashochguertel** [alvesjtiago/hover-preview#16]
With the Sublime Text 3 API you can set an max-width, max-height of the popup:

https://www.sublimetext.com/docs/3/api_reference.html#sublime.View
`show_popup(content, <flags>, <location>, <max_width>, <max_height>, <on_navigate>, <on_hide>)`

So you can modify [hover_preview.py#L120](https://github.com/alvesjtiago/hover-preview/blob/master/hover_preview.py#L120) and [hover_preview.py#L157](https://github.com/alvesjtiago/hover-preview/blob/master/hover_preview.py#L157) to somethinge like this:

```
                view.show_popup('<img style="width:' + str(width) +
                                            'px;height:' + str(height) +
                                            'px;" src="data:image/png;base64,' +
                                    encoded +
                                '">',
                                 flags=sublime.HIDE_ON_MOUSE_MOVE_AWAY,
                                 location=point,
                                 max_width=640,
                                 max_height=400)
```

```
                    view.show_popup('<img style="width:' + str(width) +
                                                'px;height:' + str(height) +
                                                'px;" src="data:image/png;base64,' +
                                        encoded +
                                    '">',
                                     flags=sublime.HIDE_ON_MOUSE_MOVE_AWAY,
                                     location=point,
                                     max_width=640,
                                     max_height=400)
```

Additional the following place in code [hover_preview.py#L15](https://github.com/alvesjtiago/hover-preview/blob/master/hover_preview.py#L15) as long I understand the python / your code.

**Comment by alvesjtiago** [comment]
Thank you for the suggestions @tobiashochguertel. I've just released version 0.0.13 which takes into account your suggestion but maximizes the popup's viewport according to the window and resizes the image accordingly. 
Please let me know what you think once you get the chance to try it and once again thanks for the feedback.

**Comment by tobiashochguertel** [comment]
@alvesjtiago That's perfect 👍 :-)

![Quick view on my System...](https://cloud.githubusercontent.com/assets/3332669/26757719/ae1c9408-48c6-11e7-9b77-b953a4b899fa.png)

Hover Preview and [ImagePaste](https://packagecontrol.io/packages/ImagePaste) makes the together a job.
